app/views/groups/_group_members.html.erb

```diff
@@ -3,13 +3,19 @@
     <div class="<%= membersCardViewer(group) %>">
       <div class="row">
         <div class="<%= membersCardViewerDetail(group) %>">
-          <i class="fa fa-user"></i>
+          <span class="groups-members-card-avatar">
+          <% if member.user.profile_picture.present? %>
+          <%= image_tag(member.user.profile_picture.url, alt: member.user.name, title: "#{member.user.name}\n#{member.user.email}") %>
```

Comment:
```suggestion
          <%= link_to image_tag(member.user.profile_picture.url, alt: member.user.name, title: "#{member.user.name}\n#{member.user.email}"), user_path(member.user.id) %>
```

Reply:
@VaibhavUpreti yah this is a good idea ... 
but for the 2nd idea ... the issue was its hard to see the pictures as they are so small so I added scaling maybe we can reduce the scaling a little bit because it's quite annoying to open full profile of user so I added this so you can quickly see who is the user maybe we can fix the styling in the scaling ... 
and for rounded images are you using border-radius ... i tried it but when there is rectangle pictures they result in oval shaped avatars or sometimes wirder shapes ... 
![image](https://user-images.githubusercontent.com/96580571/222056552-988edee4-620a-4255-a079-b6e883a6c110.png)

Reply:
yah done with the rounded picture and reduced the scaling let me know WDYT.